forage-android/src/main/java/com/joinforage/forage/android/model/StateIIN.kt

```diff
@@ -2,6 +2,10 @@ package com.joinforage.forage.android.model
 
 internal const val STATE_INN_LENGTH = 6
 
+/**
+ * A class that includes all of the possible US states that issue EBT Cards and their corresponding
+ * abbreviations.
```

Comment:
Can we add a remark that South & North Dakota-issued cards have the same IIN?

@dleis612 to correct me in case I missed something.